updates/testing.json

```diff
@@ -21,18 +21,18 @@
       }
     },
     {
-      "version": "31.20200113.2.0",
+      "version": "31.20200118.2.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "31.20200118.2.0",
+      "version": "31.20200127.2.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1579885200,
+          "start_epoch": 1580738400,
```

Comment:
Looks like that's Monday? I thought we were going to rollout testing today?

Reply:
yeah, that was the original plan. i think since it's so late in the day we might as well just make it monday for this one too. WDYT?